src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -382,6 +385,41 @@ describe("Payment", () => {
     expect(component.find(ErrorModal).props().show).toBe(true)
   })
 
+  describe("Analytics", () => {
+    it("tracks click when use shipping address checkbox transitions from checked to unchecked but not from unchecked to checked", () => {
+      const { Component, dispatch } = mockTracking(PaymentRoute)
+      const component = mount(<Component {...testProps} />)
+      // Initial state is checked
+      component
+        .find(Checkbox)
+        .at(0)
+        .simulate("click")
+      expect(dispatch).toBeCalledWith({
+        action_type: "Click",
+        subject: "use shipping address",
+        flow: "buy now",
+        type: "checkbox",
+      })
+      expect(dispatch).toHaveBeenCalledTimes(1)
+
+      dispatch.mockClear()
+
+      // State is now unchecked
+      component
+        .find(Checkbox)
+        .at(0)
+        .simulate("click")
+      expect(dispatch).not.toBeCalled()
+    })
+
+    it("triggers order:payment event on component did mount", () => {
+      const { Component, dispatch } = mockTracking(PaymentRoute)
```

Comment:
@williardx - I think your env might be misconfigured as this `dispatch` should have failed on `git push` due to `yarn type-check`. Can you verify that husky is running as expected?

Reply:
When I push, I see that Husky is running:
```
> husky - npm run -s prepush
> husky - node v8.12.0
```
So not sure what happened there 🤔

Reply:
Actually, if I manually run `yarn type-check` I don't get any errors. The only immediate difference I see is the version: mine is v1.10.1 while on Circle it's v1.9.4.

Reply:
Could have been because of a different base branch that didn't include those changes yet